py/client/pydeephaven/session.py

```diff
@@ -86,11 +152,12 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def tables(self):
         with self._r_lock:
             fields = self._fetch_fields()
-            return [field.field_name for field in fields if field.application_id == 'scope' and field.typed_ticket.type == 'Table']
+            return [field.field_name for field in fields if
+                    field.application_id == 'scope' and field.typed_ticket.type == 'Table']
 
     @property
     def grpc_metadata(self):
```

Comment:
None of these methods have type hints.  The client needs a major lookover.

Reply:
see response above.